.github/workflows/e2e-testnet.yml

```diff
@@ -0,0 +1,45 @@
+name: E2E tests / Testnet
+
+on:
+  schedule:
+    - cron: '*/20 * * * *'
+  workflow_dispatch:
+  pull_request: # TODO: remove this trigger before merging to main
+
+jobs:
+  e2e-testnet-tests:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./e2e
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.x"
+
+      # TODO: Temprorarily we use fixed version of tbtc.js. Ultimately, we would
+      # need to point to the latest deployed version.
+      - run: npm install --save-exact @keep-network/tbtc.js@0.19.4-ropsten.2
+
+      - name: Cache Global NPM Cache
```

Comment:
Do we need this step? Can't we use `actions/setup-node`'s cache?

Reply:
We can't at this moment ([action's cache does not support `package.json` located in the repo's subfolder](https://github.com/actions/setup-node/issues/275)). But it looks that the fix for the issue is in the works, so maybe we will be able to do that in near future.

Reply:
Looks we already have this one: https://github.com/keep-network/local-setup/pull/98/commits/724cca168ea801db494325fc5457f6ccea1e68d3